corehq/apps/reminders/templates/reminders/keyword_list.html

```diff
@@ -21,7 +21,11 @@ <h2>{% trans 'Manage Keywords' %}</h2>
       <i class="fa fa-plus"></i>
       {% trans 'Add Structured Keyword' %}
     </a>
-    {% include "reminders/link_keywords_modal.html" %}
+    {% if request|toggle_enabled:"LINKED_DOMAINS" %}
```

Comment:
This toggle is on for both upstream and downstream domains, but the button should only show for upstream domains, right?

Reply:
Nevermind, I must have missed the `if linked_domains` check below.